tools/python/convert_from_tensorflow.py

```diff
@@ -562,6 +638,15 @@ def generate_sub_block_op_scope_info(self):
                 if scope + '/kernel' not in self.name_node_dict and scope.split('/Tensordot')[0] + '/kernel' not in self.name_node_dict:
                     continue
                 self.dense_scope_names.add(scope.split('/Tensordot')[0])
+            elif 'batchnorm' in node.name:
```

Comment:
why not op == ''?

Reply:
node.name may be batchnorm/xxx, batchnorm_1/xxx, etc.

Reply:
*batchnor* matches